src/util/getSortPreference.js

```diff
@@ -0,0 +1,8 @@
+// util
+import {
+  getSetting
+} from '../util.js'
+
+export const getSortPreference = contextMeta => {
+  return contextMeta.sort && contextMeta.sort.length !== 0 ? Object.keys(contextMeta.sort)[0] : getSetting(['Global Sort'])[0] || 'None'
```

Comment:
Break long ternary statements into three lines logically corresponding to "if", "then", and "else".